lti_consumer/lti_xblock.py

```diff
@@ -678,6 +683,19 @@ def _get_statici18n_js_url(loader):  # pragma: no cover
                 return text_js.format(lang_code=code)
         return None
 
+    def _get_statici18n_js_url(self, loader):
+        """
+        Return the JavaScript translation file provided by the XBlockI18NService.
+        """
+        if url_getter_func := getattr(self.i18n_service, 'get_javascript_i18n_catalog_url', None):
```

Comment:
```suggestion
        if i18n_service := self.runtime.service(self, "i18n"):
            if url_getter_func := getattr(i18n_service, 'get_javascript_i18n_catalog_url', None):
```

No need for the `DummyTranslationService` imho. Keeping the code simpler is better.